specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>url</code></td>
+    <td>URL (string)</td>
+    <td>Address used to deliver events to the Addressable.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.Destination
+
+Destination is used to indicate the destination for event delivery. A
+Destination eventually resolves the supplied information to a URL by resolving
+`uri` relative to the address of `ref` (if provided). `ref` may be an
+[Addressable](#duckv1-addressable) object or a `v1/Service` Kubernetes service.
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>ref</code></td>
+    <td><a href="#duckv1kreference">duckv1.KReference</a><br/>(Optional)</td>
+    <td>An ObjectReference to an Addressable reference to deliver events to.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>uri</code></td>
+    <td>URL (string)<br/>(Optional)</td>
+    <td>A resolved URL to deliver events to.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.SubscriberSpec
+
+SubscriberSpec represents an automatically-populated extraction of information
+from a [Subscription](#subscription).
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>uid</code></td>
+    <td>UID (string)</td>
+    <td>UID is used to disambiguate Subscriptions which might be recreated.</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>generation</code></td>
+    <td>int64</td>
+    <td>Generation of the copied Subscription.</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>subscriberUri</code></td>
+    <td>URL (string)</td>
+    <td>The resolved address of the Subscription's <code>spec.subscriber</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>replyUri</code></td>
+    <td>URL (string)</td>
+    <td>The resolved address of the Subscription's <code>spec.reply</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a></td>
+    <td>The resolved Subscription delivery options. The <code>deadLetterSink</code> should use the <code>uri</code> form.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.SubscriberStatus
+
+SubscriberStatus indicates the status of programming a Subscription by a
+Channel.
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>uid</code></td>
```

Comment:
do we need to make a statement about how the list in this array MUST match what's in the spec? And the UIDs needs to match too

Reply:
shouldn't all UID be required?